doc/docs/tutorial_docker.md

```diff
@@ -38,17 +38,30 @@ your environment.
    `List/patient-list-example` resource.
 
 5. Run the FHIR Information Gateway Docker image with the `list` access checker.
+
    ```shell
    docker run \
      -e TOKEN_ISSUER=http://localhost:9080/auth/realms/test \
      -e PROXY_TO=http://localhost:8099/fhir \
      -e BACKEND_TYPE=HAPI \
      -e RUN_MODE=PROD \
      -e ACCESS_CHECKER=list \
+     -e AUDIT_EVENT_ACTIONS_CONFIG=CRUDE \
      --network=host \
      us-docker.pkg.dev/fhir-proxy-build/stable/fhir-gateway:latest
    ```
 
+   !!! tip "Docker Host Networking Note"
+
+   The `--network=host` flag is used to allow the FHIR Information Gateway
+   container to access services running on the host machine (Keycloak and HAPI
+   FHIR server in this case). This flag works on Linux hosts. If you are using
+   Docker Desktop on Windows or Mac, you may need to replace `localhost` with
+   `host.docker.internal` in the environment variable values above and remove
+   the `--network=host` flag. Alternatively for Docker Desktop versions 4.34 and
+   later, you can enable
+   [host networking](https://docs.docker.com/engine/network/tutorials/host/#prerequisites).
```

Comment:
Is this the right section of that doc that you want to link or is the following link better? https://docs.docker.com/engine/network/drivers/host/#docker-desktop